lib/client.js

```diff
@@ -250,9 +256,7 @@ function Client() {
           return cb(new Error(data)); // sometimes errors are returned as strings.
         }
       }
-
-      data.statusCode = res.statusCode;
```

Comment:
I rather leave the `statusCode` property in the `res` object and would not pollute the data in the response body.
So instead the callback will be called with the `res` object as a third argument. This means it is also downwards compatible with older versions of the package. But the benefit here is that the package user could use the `res` object to handle more information from the HTTP response.